docs/user/dashboard/lens.asciidoc

```diff
@@ -207,7 +211,10 @@ refer to <<explore-fields-in-your-data,Explore the fields in your data>>.
 [[is-it-possible-to-sort-dimensions-in-a-chart]]
 ===== How do I sort the dimensions in a chart?
 
-Sorting dimensions in visualizations is unsupported in *Lens*.
+Sorting dimensions in visualizations is unsupported in *Lens* for charts.
+
+Sorting is possible in *Lens* datatables, but it is limited to a single column. Click on the column header to sort by the column and select the sorting criteria.
+Note that if the dimension is used within the `Columns` (transposed) then all columns that belong to the same dimension will be sorted.
```

Comment:
```suggestion
If the dimension is used within the transposed `Columns`, then all columns that belong to the same dimension are sorted.
```

Reply:
Your version sounds better. I guess the only pending issue is to understand how to call the new `transposed` feature for data tables. I'll check the release notes for some reference.

Reply:
The definition of transpose is "to change the relative position, order, or sequence of; cause to change places; interchange"

I suggest change this to:

If you use the dimension in the reordered columns, then all columns that belong to the same dimension are sorted.